components/places/src/places.udl

```diff
@@ -80,6 +92,19 @@ interface PlacesConnection {
 
     [Throws=PlacesError]
     void delete_visit(string url, Timestamp timestamp);
+
+    [Throws=PlacesError]
```

Comment:
(ie, move wipe_local next to delete_everything and add a suitably sad comment here about how the distinction need to be better defined)